pix2pix:
    ckpt_path: "$(find simulation_to_real)/src/model/real/meadow_depth.pth"
    hps:  "--output_nc 6 --model pix2segdepth --ngf 128,256,256,512,512,512,1024"
    num_threads: 0
    batch_size: 1
    serial_batches: True
    no_flip: [True]
    display_id: [-1]
test: True
subscribers:
    images: "~images"
publishers:
    seg_map: "~seg_map"
    crop_map: "~crop_map"
    img_map: "~seg_img"
